test/test_datetimeWrapper.js

```diff
@@ -603,7 +603,7 @@ describe('sundial', function() {
         expect(tzs.bigFour.length).to.equal(4);
         expect(tzs.unitedStates.length).to.equal(9);
         expect(tzs.hoisted.length).to.equal(98);
-        expect(tzs.theRest.length).to.equal(448);
+        expect(tzs.theRest.length).to.equal(449);
```

Comment:
why? Maybe a little comment in the commit would help ;)

Reply:
It was the same the last time we upgrade the moment-timezone package.
The number of timezone in the https://www.iana.org/time-zones have changed...